src/main/resources/templates/success.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout.html}">
+<head>
+    <meta charset="UTF-8">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <title>Title</title>
+    <script>
+        function click() {
+            document.getElementById("link").click();
+        }
+
+        setTimeout("click()", 3000);
+    </script>
+    <style>
+        #link {
+            display: none;
+        }
+
+        body {
+            display: block;
+            margin-left: 2%;
+            margin-right: 2%;
+            width: auto;
+        }
+    </style>
```

Comment:
Wydziel style do zewnętrznego arkusza. Zamiast stosować bardzo ogólne selektory typu np. body, lepiej opakuj elementy w dodatkowego diva, który będzie miał jakąś klasę. W bootstrapie najczęściej robi się to tak, że w głównym layoucie będziesz miał
```
<div class="container">
  <div th:replace="fragments :: header"> </div>
   <div layout:fragment="content"></div>
   <div th:replace="fragments :: footer"> </div>
</div>
```
Klasa container doda marginesy po bokach